### PR TITLE
Add InnerSource SIG Co-Lead Role Description

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description
This pull request adds the following markdown files that help describe the open InnerSource SIG Co-Lead role as advertised in issue #22.

- `docs/open-roles/README.md`
- `docs/open-roles/innersource-sig-co-lead.md`

Please feedback prior to accepting the merge.

### CODE REVIEW

**Review by claredillon — COMMENTED** [review]
This looks great to me @mcleo-d . The only other suggestion is perhaps explicitly mentioning that from time the Co-Lead is asked to chair meetings, and report to the FINOS Board on progress.

**Comment by mcleo-d** [comment]
> This looks great to me @mcleo-d . The only other suggestion is perhaps explicitly mentioning that from time the Co-Lead is asked to chair meetings, and report to the FINOS Board on progress.

Thanks @claredillon - I have added the additional responsibility which can be viewed below ...

File: docs/open-roles/innersource-sig-co-lead.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- You'll help to guide overall direction, setting the priority and strategy for the SIG backlog and owning the overall direction of the special interest group. The future of the InnerSource SIG is discussed and planned on project calls and in our GitHub Issues. You’ll help drive these conversations and will guide the SIG towards the best possible outcomes.
+
+- You'll educate and drive the quality of InnerSource deliverables and documentation. You'll work to identify and implement opportunities for getting the FINOS and InnerSource Commons Communities involved in contributing solutions to the SIG.
+
+- You'll help define and uphold InnerSource SIG community standards. You will represent the SIG within the FINOS community, both with the quality of your solutions and the way you treat people in the community.
+
+- The InnerSource SIG Co-Lead is also asked to chair InnerSource SIG meetings and report progress to the FINOS board on an 'as and when' basis.
```
Comment (mcleo-d) [inline docs/open-roles/innersource-sig-co-lead.md:16]: Additional accountability added ...

- The InnerSource SIG Co-Lead is also asked to chair InnerSource SIG meetings and report progress to the FINOS board on an 'as and when' basis.

**Review by amaltson — APPROVED** [review]
LGTM

**Pull request merged**